config/routes.rb

```diff
@@ -0,0 +1,14 @@
+Rails.application.routes.draw do
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+
+  namespace :api do
+    namespace :v1 do
+      resources :users, only: [:create], path: 'signup'
+      resources :reservations
+      resources :motorcycles
```

Comment:
- A good idea to consider here, is to implement the same as users...the `only:` statement.  Why don't add it with created functions? 🐶